include/crm/common/ipc.h

```diff
@@ -154,6 +154,7 @@ enum crm_ipc_flags
 
 typedef struct crm_ipc_s crm_ipc_t;
 
+// @COMPAT max_size parameter is deprecated and unused since 3.0.1
```

Comment:
pedantically I'm not sure if it makes sense to call it deprecated since there's no way to avoid passing it; should definitely say it's unused regardless